File: Swiman/__scripts__/modules/api/models/Member.ts


    
        import { MemberMeta } from "./MemberMeta";
    

    import * as assign from "object-assign";

    

    


    class Member {
        

        
        // ID
        public id: number = 0;
        // LAST_NAME
        public last_name: string = undefined;
        // FIRST_NAME
        public first_name: string = undefined;
        // BIRTHDAY
        public birthday: Date = undefined;
        // MALE
        public male: boolean = false;
        // ADRESS
        public adress: string = undefined;
        // POSTALCODE
        public postalcode: string = undefined;
        // CITY
        public city: string = undefined;
        // METAS
        public metas: MemberMeta[] = [];

        constructor(init?: Partial<Member>){
            assign(this, init);
        }
        
    }


export {  Member }

File: Layer_datas/Services/Objects/members/Member.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Layer_datas.Services.Objects.members
{
    public class Member
    {
        public int id { get; set; }
        public string last_name { get; set; }
        public string first_name { get; set; }
        public DateTime birthday { get; set; }
        public bool male { get; set; }
        public string adress { get; set; }
        public string postalcode { get; set; }
        public string city { get; set; }

        public List<MemberMeta> metas { get; set; }
    }
}
